src/main/java/org/xwiki/git/internal/DefaultGitManager.java

```diff
@@ -67,32 +69,42 @@
     @Inject
     private Logger logger;
 
-    @Override
-    public Repository getRepository(String repositoryURI, String localDirectoryName)
+    private Repository getGitRepository(String repositoryURI, String localDirectoryName,
+                                        String username, String token)
```

Comment:
For indentations and code style please see https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/

Reply:
Updated. Please confirm.